apps/AEPSampleApp/extensions/Edge.js

```diff
@@ -0,0 +1,43 @@
+import React, {Component} from 'react';
```

Comment:
We need to have a Copyright header here.
btw, can you help to add Copyright headers for other javascript files in the same directory?